lib/v2/screens/transfer/send/send_confirmation/components/generic_transaction_success_diaog.dart

```diff
@@ -0,0 +1,166 @@
+import 'package:flutter/material.dart';
```

Comment:
A whole new success dialog for generic transactions

Reply:
did we entirely miss this dialog? 
Or was this never needed before?

Reply:
We missed it!